### ISSUE TITLE
How to use it with vue-fontawesome?

### ISSUE THREAD

**Issue by GreyXor** [invictus-codes/nuxt-vuetify#1]
### What happened?

Hello, I have followed this part https://vuetifyjs.com/en/features/icon-fonts/#font-awesome-svg-icons but I can't get it to work. Can you tell me how to use it with Font Awesome vue-fontawesome ?

### Version

v0.2.18

### What browsers are you seeing the problem on?

Firefox

### Additional context

nuxt.config.ts
```typescript
import { aliases, fa } from "vuetify/iconsets/fa-svg";

export default defineNuxtConfig({
  devtools: true,
  css: ["@fortawesome/fontawesome-svg-core/styles.css"],
  modules: [
    "@invictus.codes/nuxt-vuetify",
  ],
  vuetify: {
    vuetifyOptions: {
      icons: {
        defaultSet: "fa",
        aliases,
        sets: {
          fa,
        },
      },
    },
  },
});
```
plugins/fontawesome.ts
```typescript
import { library, config } from "@fortawesome/fontawesome-svg-core";
import { FontAwesomeIcon } from "@fortawesome/vue-fontawesome";
import { fas } from "@fortawesome/free-solid-svg-icons";

config.autoAddCss = false;

library.add(fas);

export default defineNuxtPlugin((nuxtApp) => {
  nuxtApp.vueApp.component("font-awesome-icon", FontAwesomeIcon);
});
```

**Comment by jvhellemondt** [comment]
Hi, thanks for reporting this. Are you getting any kind of error?

**Comment by GreyXor** [comment]
> Hi, thanks for reporting this. Are you getting any kind of error?

Hello thanks,
I do not receive any errors. Just no icons,

This code does not work:
 ```html
 <v-icon icon="fas fa-home" /> < ! -- I've tried `fa`, `fas` and some other -- >
```

This code works:
```html
<font-awesome-icon :icon="['fa', 'home']" />
```

I would like to find a solution to register the icons in vuetify so that I can use them as in the first code

**Comment by jvhellemondt** [comment]
I'm working on a CodeSandbox for this in my spare time. Will get back to you asap

**Comment by jvhellemondt** [comment]
I think I have found the issue. I'm using json stringify for the module options to them to the runtime (src/runtime/templates/plugin.mts#5). You pass (as you should) a function as IconSet. That doesn't work with json. Will fix and test asap

**Comment by jvhellemondt** [comment]
That wasn't it. I have created a [ticket](https://github.com/nuxt/nuxt/issues/20933) at Nuxt.

**Comment by GreyXor** [comment]
Thank you @jvhellemondt, so it's not related to `nuxt-vuetify` module but nuxt itself ?

**Comment by jvhellemondt** [comment]
No it's not with Nuxt. Nuxt is great.

It's within the module -- I'm trying to pass the iconset through some hoops, however it is lost during the build step (thus not working). I am unable to figure out how to do this, while keeping the awesome developer experience, so I asked the nuxt team how they would do it.

**Comment by GreyXor** [comment]
@jvhellemondt hello, gentle ping. https://github.com/nuxt/nuxt/issues/20933 is now closed.
Can we find a workaround or something to make [vue-fontawesome](https://github.com/FortAwesome/vue-fontawesome) work with [nuxt-vuetify](https://github.com/invictus-codes/nuxt-vuetify) ?

**Comment by GreyXor** [comment]
@jvhellemondt Hello, second gentle ping, did you manage to find a workaround or a fix ? thank you

**Comment by jvhellemondt** [comment]
Nope. I followed instructions of nuxt team however couldn't dive in entirely. For now, out of scope on this module

**Comment by GreyXor** [comment]
OK, if anyone needs a vuetify nuxt module that handle vue-fontawesome, I recommand: https://github.com/userquin/vuetify-nuxt-module. I've switched to it one month ago and working very nice
